index.html

```diff
@@ -10,19 +10,30 @@
   <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;600;700&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;700;600&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/845ab240d8.js" crossorigin="anonymous"></script>
 </head>
 <body>
   <header>
-    <nav class="navigation">
-      <a href="https://github.com/TaiwoEnoch" class="logo">Enoch</a>
-      <ul class="nav-bar">
-        <li><a href="#">Portfolio</a></li>
-        <li><a href="#">About</a></li>
-        <li><a href="#">contact</a></li>
-        <li><img src="./images/Icon-Mail.png" class="mail"></li>
-      </ul>
-      <img src="./images/Union.png" class="mobile">
-    </nav>
+    <div class="my-nav-bar">
+      <nav class="navigation">
+        <a href="https://github.com/TaiwoEnoch" class="logo">Enoch</a>
+        <ul id="navbar-links" class="nav-bar">
+          <li><a href="#my-work">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">contact</a></li>
+          <li><img src="./images/Icon-Mail.png" class="mail"></li>
+        </ul>
+          <img src="./images/Union.png"  id="toggle-button" class="mobile">
+      </nav>
+      <div id="mobile-menu" class="mobile-menu-hide">
+        <i id="mobile-menu-icon" class="fa-solid fa-x flex-end"></i>
```

Comment:
- [ ] Good job at implementing the functionality of closing the mobile menu, but Kindly 😊try to add the correct color of the ```X ``` icon to be like the Figma Design 😃
- [ ] Also consider😃 decreasing the margin between the ```X``` icon and the nav links to be like the Figma Design👌

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/215624576-98c2b9e9-700f-40db-9276-30b31451e8df.png)|   ![image](https://user-images.githubusercontent.com/83360636/215624811-0e23567a-62bf-4594-8a07-48c147c4290a.png)|